integration-test/1608-prefer-forest-labels-over-wilderness.py

```diff
@@ -0,0 +1,150 @@
+# -*- encoding: utf-8 -*-
+from . import FixtureTest
+
+
+class ForestTest(FixtureTest):
+
+    def test_lake_roosevelt_national_rec_area(self):
+        import dsl
+
+        z, x, y = (9, 87, 177)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/5908558
+            dsl.way(5908558, dsl.box_area(z, x, y, 914801107), {
+                'boundary': 'national_park',
+                'boundary:type': 'protected_area',
+                'leisure': 'nature_reserve',
+                'name': 'Lake Roosevelt National Recreation Area',
+                'operator': 'National Park Service',
+                'ownership': 'national',
+                'protect_class': '5',
+                'protection_title': 'National Recreation Area',
+                'source': 'openstreetmap.org',
+                'type': 'boundary',
+            }),
+        )
+
+        # TODO: should be national park, or nature_reserve? on the one hand,
+        # it is tagged as operated by NPS, but its name contains "recreation
+        # area", not "park"?
+        self.assert_has_feature(
+            z, x, y, 'pois', {
+                'id': 5908558,
+                'kind': 'national_park',
```

Comment:
I'd be fine with this being a `nature_reserve` instead. The US National Park Service operates many different units, but only a minority are truly national parks.